conf/messages.en

```diff
@@ -976,11 +976,30 @@ authorisedOfficialAddress.checkYourAnswersLabel = Home address
 
 # Authorised Officials Previous Address Page Messages
 # ----------------------------------------------------------
-authorisedOfficialPreviousAddress.title = Has the authorised official’s address changed in the last 12 months?
-authorisedOfficialPreviousAddress.heading = Has {0}’s address changed in the last 12 months?
-authorisedOfficialPreviousAddress.error.required = Select yes if their address has changed in the last 12 months
+isAuthorisedOfficialPreviousAddress.title = Has the authorised official’s address changed in the last 12 months?
+isAuthorisedOfficialPreviousAddress.heading = Has {0}’s address changed in the last 12 months?
+isAuthorisedOfficialPreviousAddress.error.required = Select yes if their address has changed in the last 12 months
 
-authorisedOfficialPreviousAddress.checkYourAnswersLabel = Address changed in the last 12 months
+isAuthorisedOfficialPreviousAddress.checkYourAnswersLabel = Address changed in the last 12 months
+
+# Authorised Official Previous Address Messages
+# ----------------------------------------------------------
+authorisedOfficialPreviousAddress.lookupPage.title = What is the authorised official’s previous home address? - Officials and nominees
+authorisedOfficialPreviousAddress.lookupPage.heading = What is {0}’s previous home address?
+authorisedOfficialPreviousAddress.lookupPage.manualAddressLinkText = Authorised official’s previous home address does not have a UK postcode
+
+authorisedOfficialPreviousAddress.selectPage.title = What is the authorised official’s full previous home address? - Officials and nominees
+authorisedOfficialPreviousAddress.selectPage.heading = What is {0}’s full previous home address?
+authorisedOfficialPreviousAddress.selectPage.editAddressLinkText = Enter authorised official’s full previous home address manually
+
+authorisedOfficialPreviousAddress.editPage.title = What is the authorised official’s full previous home address? - Officials and nominees
+authorisedOfficialPreviousAddress.editPage.heading = What is {0}’s full previous home address?
+
+authorisedOfficialPreviousAddress.confirmPage.title = Confirm the authorised official’s previous home address - Officials and nominees
+authorisedOfficialPreviousAddress.confirmPage.heading = Confirm {0}’s previous home address
+authorisedOfficialPreviousAddress.confirmPage.changeLinkText = Change authorised official’s previous home address
+
+authorisedOfficialPreviousAddress.checkYourAnswersLabel = Previous Home address
```

Comment:
change to `Previous home address` please (lower-case 'h', both in messages.en and messages.cy)